generators/generator-bot-enterprise-assistant/README.md

```diff
@@ -18,7 +18,7 @@ Includes support for:
 
 Use this template if you want to...
 
-- Create an enterprise assistant
+- Create a conversational experience for common enterprise scenarios (i.e. calendar management and user directory lookup)
 - See examples of more complex conversational flows, and more advanced language understanding and generation
 - See an example of a root bot and skills handling complex user interactions
```

Comment:
Should we also mention examples of what these complex user interactions are? Like interruption, card actions, etc.?